src/uct/base/uct_iface.h

```diff
@@ -142,17 +147,21 @@ enum {
 /**
  * In debug mode, if _condition is not true, generate 'Invalid length' error.
  */
-#define UCT_CHECK_LENGTH(_length, _min_length, _max_length, _name) \
+#define UCT_CHECK_LENGTH_EX(_length, _min_length, _max_length, _action_on_fail, _name) \
```

Comment:
can we avoid this windows-style naming please?

Reply:
removed at all as not needed anymore